README_template.md

```diff
@@ -0,0 +1,38 @@
+This is a skeleton project from which to create new kotlin projects from.
```

Comment:
I think we don't need to keep this as we can refer to it in [the template repository](https://github.com/ministryofjustice/hmpps-template-kotlin#readme).